cmd/utils/flags.go

```diff
@@ -740,7 +740,7 @@ var (
 	VTRecoveryIntervalFlag = cli.Uint64Flag{
 		Name:  "vtrecoveryinterval",
 		Usage: "Set the value transfer recovery interval (seconds)",
-		Value: 60,
```

Comment:
We need to check why this value is set before.
@KimKyungup Could you please check this change is okay based on your knowledge?

Reply:
I'm sorry, I omitted why this change was created. Let me put it here.

The recovery logic has a trade-off between performance overhead and fast recovery about failed Value Transfer Request (VTR). 
The recovery for failed VTR is executed by a given interval, which means that the failed unhandled event would be inserted into the event pool as fast as the interval is low. However, we should call that logic often, which brings performance overhead. The overhead is a linear search of finding unhandled events from block number A to the current block number. The block number A is a number that contains the latest handled VTR. The worst case for this overhead depends on the range between A and the current block number. I thought 60 secs seems late to find unhandled VTR and it leads to make bad user experience. [See the TODO list](https://github.com/klaytn/klaytn/pull/1271#issue-1178117946). I loved to hear an opinion from experienced whoever.

Reply:
@hyunsooda Could you please add the TODO list somewhere in the source code or a separate issue? I think it is a more traceable way of TODO listing.

And one more thing is that, do we have a logic that can prohibit the execution if the prior execution is not ended?

Reply:
There's no guard for interleaving execution. Let me first describe the implication of these lacks.

The unhandled events can be inserted into the event pool again multiple times by unintentionally calling that functions. However, only the first event would be correctly handled and the other ones are reverted since the nonce written in the event is invalid.

The current recovery logic is not concurrently called. Say recovery function takes 10s to finish and the given interval is 5s, then the recovery function is called every 10s, not 5s.

Reply:
okay. got it. I understood that the function is not concurrently executed.